### ISSUE TITLE
[Feature Request] Lazy delete toggle permanently on - Signature list

### ISSUE THREAD

**Issue by dnolen** [exodus4d/pathfinder#860]
Request. Signature list change option to lazy delete so it is permanently on for until toggled on. 

Current behaviour turns the lazy delete option off once the entreies are updates from the system clipboard.

**Comment by Slivo-fr** [comment]
This was the previous behavior.

It's been changed because it cause many mistakes iirc. 
I'd rather keep it like it is now if we can't have an option to toggle the behavior.

**Comment by dagoaty** [comment]
I'd vote for it to have a sticky state per session. i.e. if you toggle it on it remains on until you are logged out.
An 'undo' option would cover accidentally blowing away sigs but I imagine that'd be a lot more work.

**Comment by Slivo-fr** [comment]
I disagree with this. Hope it stay as it is and you get an option to change the behavior

**Comment by CalMatar** [comment]
If you look at the changes he is making this is a moot point as he has a change in progress that allows you to see what changes are being made before things are committed and able to be rolled back (I think).

**Comment by dnolen** [comment]
> I disagree with this. Hope it stay as it is and you get an option to change the behavior

I'd rather have the option to toggle the behaviour, but the undo option has also been mentioned before. In terms of creating that I am not sure of the complexity but it has been an option in other mappers that is rather useful.

**Comment by exodus4d** [comment]
It is not planned to revert the new behavior. 
As @Slivo-fr said, it caused many mistakes by users who forgot to select the proper system before pasting and _fucked up_ the scanning work of others. (even if there is now a roll back function for signatures)

But I would hold the discussion about it open to see what others think about it.

Suggestion: What would you think about a new shortcut that toggles the function temporary?
- No need to scroll down for activation each time
- Prevents some mistakes where users accidentally pressed `ctrl`+`v` and forgot about the active browser window
- If _hopefully_ makes users think one more time about the consequences.

**Comment by dnolen** [comment]
I'm in favour of the toggle option personally. It can be set defaulted to the current behaviour for the map but if per user basis someone wanted to toggle it on for a session that would just reduce the requirement to keep toggling it back on.

**Comment by porowns** [comment]
No change needed, tbh.
